pkgs/os-specific/linux/prl-tools/default.nix

```diff
@@ -1,48 +1,55 @@
 { stdenv, lib, makeWrapper, p7zip
 , gawk, util-linux, xorg, glib, dbus-glib, zlib
 , kernel ? null, libsOnly ? false
-, undmg, fetchurl
+, fetchurl, undmg, perl, autoPatchelfHook
 }:
 
 assert (!libsOnly) -> kernel != null;
 
-let xorgFullVer = lib.getVersion xorg.xorgserver;
-    xorgVer = lib.versions.majorMinor xorgFullVer;
-    x64 = if stdenv.hostPlatform.system == "x86_64-linux" then true
-          else if stdenv.hostPlatform.system == "i686-linux" then false
-          else throw "Parallels Tools for Linux only support {x86-64,i686}-linux targets";
+let
+  aarch64 = stdenv.hostPlatform.system == "aarch64-linux";
+  x86_64 = stdenv.hostPlatform.system == "x86_64-linux";
+  i686 = stdenv.hostPlatform.system == "i686-linux";
+  _ = if (aarch64 || x86_64 || i686) then true
```

Comment:
This won't fire at all, because the language is lazy. You could use `assert aarch64 || x86_64 || i686` after `in` instead.